flash_patcher/patcher.py

```diff
@@ -1,4 +1,6 @@
 from importlib.metadata import PackageNotFoundError, version
+# pylint: disable=no-name-in-module
```

Comment:
Why is this needed?

Reply:
Pylint complains about `from logging import DEBUG`. I'm not entirely sure why.

I also tried this:
```
import logging
logger.setLevel(logging.DEBUG)
```

but here, pylint complains about `E1101: Module 'logging' has no 'DEBUG' member (no-member)`

I think we should keep the import as is since all of our other imports use `from` statements